tools/unit-tests/unit-delta.c

```diff
@@ -0,0 +1,160 @@
+#include <check.h>
```

Comment:
License header?

Reply:
it's missing in  a few unit tests file. Adding.